src/main/java/org/example/homework10/splitTheSentenceLambda/Lambda.java

```diff
@@ -0,0 +1,15 @@
+package org.example.homework10.splitTheSentenceLambda;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Lambda {
+    public static void main(String[] args) {
+        String sentense = "we chose items at random";
+        splitAndPrint(sentense);
+    }
+    public static void splitAndPrint(String sentence){
+        var splitSentense = new ArrayList<>(Arrays.asList(sentence.split(" ")));
+        splitSentense.forEach(System.out::println);
```

Comment:
minor:
```suggestion
        Arrays.asList(sentence.split(" ")).splitSentense.forEach(System.out::println);
```
you can write it even in one line

Reply:
I tried to create a function as you wrote, but it doesn't compile for me